tests/installation/first_boot.pm

```diff
@@ -59,10 +59,7 @@ sub run {
         handle_login;
     }
 
-    my @tags = qw(generic-desktop);
-    if (check_var('DESKTOP', 'kde') && get_var('VERSION', '') =~ /^1[23]/) {
-        push(@tags, 'kde-greeter');
-    }
```

Comment:
same here

Reply:
kde-greeter is the defunct, kde-specific precursor of opensuse-welcome. So IMHO it is not unrelated.

Reply:
ok, fine